debian/copyright

```diff
@@ -0,0 +1,32 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
this file needs some work and currently is not correct. I'd suggest removing the lines 5-16 and instead write something like Copyright: see https://github.com/UltraStar-Deluxe/USDX

Reply:
This is a machine readable file which needs to strictly follow the format given on https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/. It is a prerequisite to get this file right before asking the Debian FTP masters to include an UltraStar Deluxe package into the distribution. While we currently don't strive to do that I think we nevertheless should invest some time here to get this file right. Maybe someone could help my with this task.

Can you name the problems you found here to help me fix these?

PS: I don't insist in a particular copyright for the files in the debian directory. I would like to put these files under the same license which is used for the rest of the project.

Reply:
as far as what I read in the copyright format document, replacing lines 6 to 16 by "Copyright: see https://github.com/UltraStar-Deluxe/USDX" would be compliant and be just fine.

Reply:
I think changing the line starting with "Copyright:" in the file is ok:

Files: *
Copyright: 2015 - 2017 see https://github.com/UltraStar-Deluxe/USDX
License: GPL-2.0+

Files: src/lib/SDL2/* src/lib/dglOpenGL/*
Copyright: 2015 - 2017 see https://github.com/UltraStar-Deluxe/USDX
License: MPL-2.0

...

The block for the files in the debian directory can be omitted because I will use the same license like most of the rest of the code there.

But  the source in the src/lib/SDL2 directory is under the MPL-2.0 license, same for src/lib/dglOpenGL. So this block is needed inside the copyright file. When we extent the file to get every used license right, then it should be rather easy to get an UltraStar DX package inside the Debian and/or Ubuntu repository sometimes in the future.

Reply:
please remove the year numbers in the copyright lines, as they are of no use in most countries (and also not correct).
On a side note, if every license for every part of code that was included needs to be mentioned in this debian/copyright file, then you still have a bit of work left to do ;-) See the files in this repository containing "LICENSE" in their name.